python/python/glide/glide_client.py

```
@@ -134,6 +134,13 @@ def __del__(self) -> None:
             if "no running event loop" in str(e):
                 # event loop already closed
                 pass
+        try:
+            # Safety mechanism to clean up resources on the Rust core side by attempting
+            # to close the UDS if close() wasn't properly called by the user.
+            self._writer.close()
+        except Exception:
+            # Ignore any exceptions that occur during the writer close operation.
```

why do we silence exceptions in that case?